### DISCUSSION TITLE
Badics

### DISCUSSION THREAD

**Discussion by gd12356** [jef/streetmerchant#1041] [Help]
I have installed Street merchant and can run it.

But what file do I edit to remove the products already in the run  file and add my own.

Is there a guide for creating or updating the currently used search file?

Thanks

**Comment by Ben7530** [comment]
It's all in the Wiki... https://github.com/jef/streetmerchant#customization

If you're on a mac, make sure to show hidden folders / files. The reason why Jef says to edit and rename .env-example, because it's easier than saving a file as only an extension on certain platforms.

**Comment by gd12356** [comment]
Thanks for replying.

I did as instructed but it still comes back with pings for 3070, 3080 3090 even if I remove them from the .env?
